tests/test_glean.py

```diff
@@ -67,11 +67,17 @@ def test_generic_schema(self, glean, config):
 
         final_schemas = {k: schemas[k][0].schema for k in schemas}
         for name, schema in final_schemas.items():
+            meta = schema.pop("mozPipelineMetadata")
```

Comment:
`print_and_test` makes missing/altered elements in JSON much easier to understand on failure. Can We instead explicitly update the `generic_schema` to include this section?

Reply:
I'll look into whether we can refactor this.

Reply:
Just to be clear - I don't see a great way to update the class function. Instead just add this blob to `generic_schema` in this test, and compare it all using `print_and_test`.